libcxx/test/std/ranges/range.utility/range.utility.conv/to.pass.cpp

```diff
@@ -119,6 +120,7 @@ struct Fallback {
   constexpr void push_back(value_type) {}
   constexpr value_type* begin() { return &x; }
   constexpr value_type* end() { return &x; }
+  std::size_t size() const { return 0; }
```

Comment:
IMO this should be constexpr too. I don't feel to strongly about it when backporting it, but please fix this in main.